oak_restricted_kernel/src/mm/bitmap_frame_allocator.rs

```diff
@@ -100,9 +100,9 @@ impl<S: PageSize, const N: usize> BitmapAllocator<S, N> {
             })
             .max_by_key(|(start_idx, end_idx)| end_idx - start_idx)
             .map(|(start_idx, end_idx)| {
-                PhysFrame::range_inclusive(
+                PhysFrame::range(
                     self.frame(start_idx).unwrap(),
-                    self.frame(end_idx).unwrap(),
+                    self.frame(end_idx).unwrap() + 1,
```

Comment:
Isn't there a way to also treat `end_idx` itself as exclusive, elsewhere?

Reply:
The particular algorithm I've implemented here comes up with an inclusive answer. In fact, this is why I originally used inclusive ranges to begin with.

However, (a) inclusive ranges have proved to be a pain elsewhere so it makes sense to concentrate it here, and (b) I suspect this whole `fn` will go away in not too distant future so I'm inclined to leave it as is.